packages/lambda-powertools-middleware-correlation-ids/__tests__/index.js

```diff
@@ -202,6 +211,25 @@ const standardTests = (genEvent) => {
       })
     })
   })
+
+  describe('when call-chain-length is provided in the event', () => {
+    it('increments it by 1', () => {
+      const id = uuid()
+
+      const correlationIds = {
+        'x-correlation-id': id,
+        'call-chain-length': 1
```

Comment:
might be worth a test to ensure it supports string call chain length values
```js
        'call-chain-length': '1'
```

Reply:
how so?

Reply:
have merged the PR for now, I don't think we need to support string call-chain-length value as it's only in `string` during "transport", which is already covered by the tests when we turn correlation IDs (`call-chain-length` in `int`) to SQS events (turns it into a `string` message attribute).